internal/renderers/femtovg/lib.rs

```diff
@@ -153,19 +148,20 @@ impl FemtoVGRenderer {
     /// Render the scene using OpenGL. This function assumes that the context is current.
     pub fn render(
         &self,
+        window: &i_slint_core::api::Window,
```

Comment:
Will it be a bug if the user start passing a different window with a possibly different component?

Reply:
I'd say no, it shouldn't be. I think we should move towards the ability to keep a renderer but change the content it renders.